docs/source/harvesters/scanr.rst

```diff
@@ -1,16 +1,16 @@
 Harvesters : ScanR
 =====================================
 
-Data source
------------
-
-SoVisu+ ScanR Harvester fetches data from the latest ScanR API.
-
 .. note:: The ScanR Harvester requires identifiers and credentials from the ScanR API, currently under development and not yet released to the public.
     Obtain these by contacting the ScanR team.
     They should be integrated into the harvester's environment variables.
```

Comment:
We should give more information (which variables)

Reply:
Environment variables are now mentioned in scanr.rst. The same issue has been solved in scopus.rst